### DISCUSSION TITLE
Request for support of RM67162 device used on the ESP32-T-Display (536 x 240) with the AMOLED display

### DISCUSSION THREAD

**Discussion by rpbgit** [Bodmer/TFT_eSPI#3182] [New processor/display support or capability request]
seems these devices are getting popular.   the LilyGo T-Display-S3 (536 x 240) with the AMOLED has been out for quite a while, and all my other projects have been using the T-Display-S3 (320 x 170) 1.91" which is fully supported by the TFT_eSPI lib.   LVGL is an atomic powerered swiss army knife.   could you put this into the work Q ??  saw a discussion about a year ago on similar topic.  ref:    https://github.com/Xinyuan-LilyGO/LilyGo-AMOLED-Series

**Comment by twj42** [comment]
I see the library supports the _RM68120 (support files added but untested) and RM68140_, does anyone know if these devices are compatible with the RM67162 ?

**Comment by byte-for-byte** [comment]
I second this proposal. Are there actually plans to support this device?
